### PR TITLE
[W4.11e][W17-3]Koh Zheng Wei

### PR DESCRIPTION
**Pull request by Zenious**
Test Cases created for Utils#isAnyNull

### CODE REVIEW

File: test/java/seedu/addressbook/util/TestUtil.java [hunk 1]
```diff
@@ -24,7 +22,28 @@
 import seedu.addressbook.data.person.ReadOnlyPerson;
 import seedu.addressbook.data.person.UniquePersonList.DuplicatePersonException;
 
+import static org.junit.Assert.*;
```
Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/util/TestUtil.java]: Our coding standard does not allow using wildcards for imports.

File: test/java/seedu/addressbook/util/TestUtil.java [hunk 2]
```diff
@@ -24,7 +22,28 @@
...
+    public void isAnyNull() {
+        final Object nullObj = null;
+        final Object testObj = "test";
+        
+        // Test with one null Object
+        assertTrue(Utils.isAnyNull(nullObj));
+        
+        // Test with list of Objects including null object
+        assertTrue(Utils.isAnyNull(testObj, nullObj));
+
```
Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/util/TestUtil.java:42]: May be include tests with more than a single null object?

File: test/java/seedu/addressbook/util/TestUtil.java [hunk 3]
```diff
@@ -24,7 +22,28 @@
...
+        
+        // Test with one null Object
+        assertTrue(Utils.isAnyNull(nullObj));
+        
+        // Test with list of Objects including null object
+        assertTrue(Utils.isAnyNull(testObj, nullObj));
+        
+        // Test with non-null Object
+        assertFalse(Utils.isAnyNull(testObj));
+
```
Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/util/TestUtil.java:45]: May be include tests with more than a single non-null object?

**Review by parvathy-sudhir-pillai — COMMENTED** [review]
Good effort @Zenious. Please close this PR after reading the review comments.
